docker/config_template.yaml

```diff
@@ -190,10 +190,10 @@ archival:
 namespaceDefaults:
   archival:
     history:
-      state: "enabled"
+      state: "disabled"
```

Comment:
Can you file a separate issue to make this docker parameter which users can control?

Reply:
Created a ticket #623 Add control of archival feature initial state through docker-compose parameter.

Does this mean that the variable will override the archival state set in defaultNamespace of temporal config?